prow/plugins/milestone/milestone.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package setmilestone implements the `/milestone` command which allows members of the milestone
+// maintainers team to specify a milestone to be applied to an Issue or PR.
+package milestone
+
+import (
+	"bytes"
+	"fmt"
+	"regexp"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/pluginhelp"
+	"k8s.io/test-infra/prow/plugins"
+)
+
+const pluginName = "milestone"
+
+var (
+	milestoneRegex   = regexp.MustCompile(`(?m)^/milestone\s+(.+)$`)
+	mustBeSigLead    = "You must be a member of the [%s](https://github.com/orgs/kubernetes/teams/%s/members) github team to set the milestone."
```

Comment:
Sorry, `org` here must also be configurable (or, since we don't do anything with it other than the comment link maybe we can figure it out given a token in `hook` itself?)